astor/code_gen.py

```diff
@@ -128,13 +133,45 @@ class SourceGenerator(ExplicitNodeVisitor):
     using_unicode_literals = False
 
     def __init__(self, indent_with, add_line_information=False,
-                 pretty_string=pretty_string):
+                 pretty_string=pretty_string, len=len,
+                 isinstance=isinstance, callable=callable):
         self.result = []
         self.indent_with = indent_with
         self.add_line_information = add_line_information
         self.indentation = 0
         self.new_lines = 0
+        self.colinfo = 0, 0
```

Comment:
Maintaining this value makes the string prettifier much faster.  It was being repeatedly calculated.